### DISCUSSION TITLE
Possible to prevent creation of the ~/.emacs.d/snippets directory

### DISCUSSION THREAD

**Discussion by doolio** [emacscollective/no-littering#235] [Support] [answered]
`no-littering` creates `~/.emacs.d/etc/yasnippet/snippets` but I also see a directory created at `~/.emacs.d/snippets` presumably by `yasnippet`. Is it possible to not have the `~/.emacs.d/snippets` directory created?

In my config, `no-littering` is loaded very early on and `'yasnippet` much later by hooking into the `after-init-hook`.

```emacs-lisp
(use-package no-littering
  :demand t)

(use-package yasnippet
  :hook (after-init-hook . yas-global-mode)
  :config (setq yas-verbosity 2))
```

I appreciate this may be an issue for `yasnippet` rather than `no-littering` but I thought it best to ask here first.

**Comment by tarsius** [comment, marked as answer]
It might also be an issue with sometimes using `emacs -Q` but then somehow ending up loading `yasnippet` and that then creating the (unthemed) directory. It also happens with certain built-in functionality (which is "always" loaded) when using `emacs -Q`.

There isn't anything `no-littering` can do about it.
